### PR TITLE
use general-task image and install jdk

### PR DESCRIPTION
**Pull request by dandersonsw**
## Changes proposed in this pull request:
- Use general-task image with jdk installed for building java app

## security considerations
Switch to using hardened image

### CODE REVIEW

File: java-see/build-java-see.sh [hunk 1]
```diff
@@ -4,6 +4,15 @@ set -ex
 
 export TERM=${TERM:-dumb}
 
+wget https://download.oracle.com/java/22/latest/jdk-22_linux-x64_bin.tar.gz
```
Comment (markdboyd) [inline java-see/build-java-see.sh:7]: Doesn't the Java buildpack come with the Open JDK? I'm confused as to why this is necessary

Comment (markdboyd) [reply java-see/build-java-see.sh:7]: Oh, so this is running in CI, not CF

Comment (markdboyd) [reply java-see/build-java-see.sh:7]: And I assume we don't want to add Java to the general-task

Comment (dandersonsw) [reply java-see/build-java-see.sh:7]: Yep, that is correct on both counts

**Review by markdboyd — APPROVED** [review]

**Pull request merged**